x-pack/plugins/security_solution/public/flyout/entity_details/components/user_details_body.tsx

```diff
@@ -0,0 +1,155 @@
+import { getUsersDetailsUrl } from '../../../common/components/link_to/redirect_to_users';
+import type {
+  ManagedUserData,
+  ObservedUserData,
+} from '../../../timelines/components/side_panel/new_user_detail/types';
+import { ManagedUser } from '../../../timelines/components/side_panel/new_user_detail/managed_user';
+import { ObservedUser } from '../../../timelines/components/side_panel/new_user_detail/observed_user';
+import * as i18n from './translations';
+
+import type { RiskScoreEntity } from '../../../../common/search_strategy';
+import { SecuritySolutionLinkAnchor } from '../../../common/components/links';
+import type { RiskScoreState } from '../../../explore/containers/risk_score';
+import { PreferenceFormattedDate } from '../../../common/components/formatted_date';
+import { RiskSummary } from './risk_summary';
+
+export const QUERY_ID = 'usersDetailsQuery';
+
+interface UserDetailsBodyComponentProps {
+  userName: string;
+  observedUser: ObservedUserData;
+  managedUser: ManagedUserData;
+  riskScoreState: RiskScoreState<RiskScoreEntity.user>;
+  contextID: string;
+  scopeId: string;
+  isDraggable: boolean;
+}
+
+export const UserDetailsBody = ({
+  userName,
+  observedUser,
+  managedUser,
+  riskScoreState,
+  contextID,
+  scopeId,
+  isDraggable,
+}: UserDetailsBodyComponentProps) => {
+  const { euiTheme } = useEuiTheme();
+
+  const lastSeenDate = useMemo(
+    () =>
+      max([observedUser.lastSeen, managedUser.lastSeen].map((el) => el.date && new Date(el.date))),
+    [managedUser.lastSeen, observedUser.lastSeen]
+  );
+
+  return (
+    <>
+      <EuiPanel hasShadow={false}>
+        <EuiFlexGroup gutterSize="s" responsive={false} direction="column">
+          <EuiFlexItem grow={false}>
+            <EuiText size="xs" data-test-subj={'user-details-body-lastSeen'}>
+              {lastSeenDate && <PreferenceFormattedDate value={lastSeenDate} />}
+              <EuiSpacer size="xs" />
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
+              <EuiFlexItem grow={false}>
+                <EuiIcon type="user" size="l" color="primary" />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiTitle size="s">
+                  <h2>
+                    <SecuritySolutionLinkAnchor
+                      deepLinkId={SecurityPageName.users}
+                      path={getUsersDetailsUrl(userName)}
+                      target={'_blank'}
+                      css={css`
+                        font-weight: ${euiTheme.font.weight.bold};
+                      `}
+                    >
+                      {userName}
+                    </SecuritySolutionLinkAnchor>
+                  </h2>
+                </EuiTitle>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
```

Comment:
instead of using a `EuiFlexGroup` here, why not using the new [FlyoutHeader](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/shared/components/flyout_header.tsx) component and put these few `EuiFlexItem` in it?

Reply:
I couldn't get rid of the `EuiFlexGroup`, but I managed to simplify the component by using `FlyoutHeader` and deleting `EuiPanel` and  `EuiHorizontalRule`.
Thank you!